resources/assets/js/annotations/annotatorContainer.vue

```diff
@@ -86,6 +86,9 @@ export default {
             userUpdatedVolareResolution: false,
             userId: null,
             crossOriginError: false,
+            // Toggles the annotation container view for images that could not be loaded,
+            // see https://github.com/biigle/core/issues/558.
+            showImage: true
```

Comment:
I think I'd like a variable named `imageLoadingError` better (similar to the existing `crossOriginError`, `false` by default). This describes better what the variable is for.